### PR TITLE
Konstantin_Ivanov

### PR DESCRIPTION
**Pull request by kostya23azaza**

### CODE REVIEW

**Comment by NikolaevArtem** [comment]
Please merge with master

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Java Core June 2021
+=====================
+Konstantin Ivanov
```
Comment (NikolaevArtem) [inline README.md]: Please merge with master

File: src/main/java/homework_1/Main.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package homework_1;
+
+public class Main {
+    public static void main(String[] args) {
+        for (String arg : args) {
+            if (arg.equals("ошибка")) {
+                System.err.println("ошибка");
```
Comment (NikolaevArtem) [inline src/main/java/homework_1/Main.java]: Doesn't work as expected. Moreover, the message should be Тревога or Alarm

File: README.md (hunk 1 above)
Comment (NikolaevArtem) [reply README.md]: Ping ping

File: src/main/java/homework_1/Main.java (hunk 2 above)
Comment (NikolaevArtem) [reply src/main/java/homework_1/Main.java]: Please read description in Homework chat again

Comment (heptaber) [reply src/main/java/homework_1/Main.java]: You need to output "Alarm!" before interruption.

**Comment by NikolaevArtem** [comment]
Please resolve all issues before putting label readyForReview. If needed, contact me or other students

**Pull request closed without merging**